### PR TITLE
Add Question Mapper/Model/Controller/ControllerTest

### PR DESCRIPTION
**Pull request by HollowPrincess**
https://github.com/malinink/easy-tests/issues/332

- [x] Add Question Mapper
- [x] Add Question Model
- [x] Add Question Controller
- [x] Add Question ControllerTest
- [x] Add Answer Mapper
- [x] Add Answer Model

### CODE REVIEW

**Comment by HollowPrincess** [comment]
@malinink 
Do we need to do Answers Model and Mapper?
All for questions was done

**Comment by HollowPrincess** [comment]
@malinink all done, coverage decreased=(

File: src/main/java/easytests/api/v1/models/Question.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public class Question {
+    private Integer id;
+
+    private String text;
+
+    private Integer type;
+
+    private Identity topic;
+
+    private List<Identity> answers;
```
Comment (malinink) [inline src/main/java/easytests/api/v1/models/Question.java]: adminAnswers not Identity

**Review by malinink — CHANGES_REQUESTED** [review]

**Comment by HollowPrincess** [comment]
@malinink fixed

**Pull request merged**